### PR TITLE
v0.2.1

### PR DESCRIPTION
**Pull request by geohacker**
This release addresses performance issues reported in #175 along with streamlining our bundle versioning. More info in the changelog.

### CODE REVIEW

File: app/screens/Explore.js [hunk 1]
```diff
@@ -186,10 +191,10 @@ class Explore extends React.Component {
     console.log('onDidFailLoadingMap', err)
   }
 
-  onRegionIsChanging = async evt => {
-    // update the redux state with the bbox
-    this.props.updateVisibleBounds(await this.mapRef.getVisibleBounds(), await this.mapRef.getZoom())
-  }
+  // onRegionIsChanging = async evt => {
```
Comment (batpad) [inline app/screens/Explore.js]: @geohacker I think we can probably remove these commented out lines

File: app/screens/Explore.js [hunk 2]
```diff
@@ -232,10 +243,17 @@ class Explore extends React.Component {
 
   async loadFeaturesAtPoint (rect) {
     try {
+      // console.time('query')
```
Comment (batpad) [inline app/screens/Explore.js]: oops, should clean this up.

File: app/screens/Explore.js [hunk 3]
```diff
@@ -232,10 +243,17 @@ class Explore extends React.Component {
...
       const { features } = await this.mapRef.queryRenderedFeaturesInRect(rect, null, this.state.clickableLayers)
+      // console.timeEnd('query')
```
Comment (batpad) [inline app/screens/Explore.js]: this as well

File: app/screens/Explore.js (hunk 1 above)
Comment (geohacker) [reply app/screens/Explore.js]: Let's leave it there, I think there's some part of me unsure about the map data fetch behaviour.

**Pull request merged**